app/models/rental.rb

```diff
@@ -0,0 +1,42 @@
+class Rental < ApplicationRecord
+  belongs_to :customer
+  belongs_to :video
+
+  validates :video_id, presence: true
+  validates :customer_id, presence: true
+  
+  def set_due_date
+   self.due_date = DateTime.now + 1.week
+  end
+  def set_videos_checked_out_count
+    self.customer.checked_out_video
+    self.videos_checked_out_count = self.customer.videos_checked_out_count
+  end
```

Comment:
considering the videos_checked_out_count in rental mimics the value from the customer model, it would be safer and less work to not bother having such a value on the rental model.
This means you don't need so much code in the rental model and reduces the risk that they become mismatched somehow, introducing a bug.